analysis/test/hh/external_library_recipe.sh

```diff
@@ -0,0 +1,29 @@
+cd $CMSSW_BASE/src/nano/
+wget http://www.hep.phy.cam.ac.uk/~lester/dtm662/mt2/Releases/oxbridgekinetics.tar.gz
+tar -xzvf oxbridgekinetics.tar.gz
+rm oxbridgekinetics.tar.gz
+mv oxbridgekinetics-1.2/ oxbridgekinetics
+cd oxbridgekinetics
+touch BuildFile.xml
+#####################################################################
+#make a buildfile for integrating oxbridgekinetics library into nano
+#specify flags in BuildFile to ignore unnecessary warnings and errors
+cat > BuildFile.xml << EOL
+<use name="root" />
+<flags LDFLAGS="-lMinuit2" />
+<flags CXXFLAGS="-Wno-error=unused-but-set-variable" />
+<flags CXXFLAGS="-Wno-error=unused-variable" />
+<flags CXXFLAGS="-Wno-error=sign-compare" />
+<flags CXXFLAGS="-Wno-error=maybe-uninitialized" />
+<export>
```

Comment:
haha i dont like these error ignores

Reply:
Ian has put them though.. should I remove them?

Reply:
Unfortunately, you'd have to rewrite the external packages to not throw them. I figured it was easier to just ignore... If we put the packages in external I guess we can have them there only and not for our code.

Reply:
> Ian has put them though.. should I remove them?

You can try to remove them. From memory it was the oxbridge packages that cause it

Reply:
yep, i dont mind the unused variables, but i think the other two should be fixed in their code